plugins/modules/azure_rm_access_token_info.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2023 Patrick Uiterwijk <@puiterwijk>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_access_token_info
+
+version_added: "1.19.0"
+
+short_description: Get Azure API access token
+
+description:
+    - Get an access token for Azure APIs.
+
+options:
+    scopes:
+        description:
+            - The scopes to request.
+        type: list
+        elements: str
+        required: True
```

Comment:
Argument 'scopes' in argument_spec is not required, but is documented as being required
```suggestion

```